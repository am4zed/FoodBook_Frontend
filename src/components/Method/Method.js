import React from 'react';
import './style.css';

const Method = () => {

  return(
    <>
      <li>{props.method}</li>
    </>
  )
};

export default Method;
