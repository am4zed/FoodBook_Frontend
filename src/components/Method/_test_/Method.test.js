import React from 'react';
import Method from '../Method';

describe("Method", () => {

  void test("snapshot", () => {
    const wrapper = shallow(<Method/>);
    expect(wrapper).toMatchSnapshot()
  });

})
