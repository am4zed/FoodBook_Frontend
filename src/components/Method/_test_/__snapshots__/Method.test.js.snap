// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Method snapshot 1`] = `
<Fragment>
  <li />
</Fragment>
`;
